ghost/email-service/test/email-renderer.test.js

```
@@ -2690,4 +2690,197 @@ describe('Email renderer', function () {
+    describe('additional i18n tests', function () {
+        let renderedPost;
+        let postUrl = 'http://example.com';
+        let customSettings = {};
+        let emailRenderer;
+        let addTrackingToUrlStub;
+        let labsEnabled;
+
+        beforeEach(function () {
+            renderedPost = '<p>Lexical Test</p><img class="is-light-background" src="test-dark" /><img class="is-dark-background" src="test-light" />';
+            labsEnabled = true; // TODO: odd default because it means we're testing the unused email-customization template
+            basePost = {
```

ESLint

[error] 'basePost' is not defined.

(no-undef)